lib/procore-sift.rb

```diff
@@ -32,7 +32,7 @@ def filters_valid?
   end
 
   def filter_errors
-    filter_validator.errors.messages
```

Comment:
The one drawback is that we've broken the contract published in the [README on the homepage](https://github.com/procore/sift#developer-usage)
```
def render_filter_errors
  render json: { errors: filter_errors }, status: :bad_request && return
end
```

The README should be updated to show the correct syntax for the 1.x version of the code (once determined), as well as the legacy syntax--since people with questions often come and give a quick look at the README (or even have it linked in some other doc) so it should be able to inform them of both the current and past valid syntax.

However, in reflecting on #lessthanjacob's comments, as well as the [other statement in the README](https://github.com/procore/sift#without-rails) that suggests we are trying to move away from a Rails dependency, I think the better course of action would be to leave this interface as-is, which would mean adding code to gracefully transform Rails 7 errors into the expected hash format .
Then, for accessing the raw errors--assuming that is one of the objectives--create a new public accessor that provides that. 
`def raw_filter_errors` or `def active_model_filter_errors` 
or something like that.
In this way we maintain current behavior while allowing for new access to the underlaying error objects.
And, in version 2.0, when maybe we drop Rails dependancies, we can consider dropping the raw object access (or maybe making it a wrapper object that quacks like an ActiveModel::Error object).